components/tools/OmeroWeb/omeroweb/webclient/custom_forms.py

```diff
@@ -129,32 +142,40 @@ def __iter__(self):
         if self.empty_label is not None:
             yield (u"", self.empty_label)
         for obj in self.queryset:
-            textValue = None            
+            textValue = None
             if isinstance(obj._obj, FileAnnotationI):
-                textValue = (len(obj.getFileName()) < 45) and (obj.getFileName()) or (obj.getFileName()[:42]+"...")
+                textValue = (
+                    (len(obj.getFileName()) < 45) and (obj.getFileName()) or
+                    (obj.getFileName()[:42]+"..."))
             elif isinstance(obj._obj, TagAnnotationI):
                 if obj.textValue is not None:
-                    if obj.ns is not None and obj.ns!="":
-                        textValue = (len(obj.textValue) < 45) and ("%s (tagset)" % obj.textValue) or ("%s (tagset)" % obj.textValue[:42]+"...")
+                    if obj.ns is not None and obj.ns != "":
+                        textValue = (
+                            (len(obj.textValue) < 45) and
+                            ("%s (tagset)" % obj.textValue) or
+                            ("%s (tagset)" % obj.textValue[:42]+"..."))
```

Comment:
that is not a bug although we could replace all ```a and b or c``` with ```a if b else c``` because of https://www.python.org/dev/peps/pep-0308/